src/main/java/de/zalando/aruha/nakadi/repository/InMemoryTopicRepository.java

```diff
@@ -0,0 +1,113 @@
+package de.zalando.aruha.nakadi.repository;
+
+import de.zalando.aruha.nakadi.NakadiException;
+import de.zalando.aruha.nakadi.domain.Topic;
+import de.zalando.aruha.nakadi.domain.TopicPartition;
+
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.stream.Collectors.toList;
+
+public class InMemoryTopicRepository implements TopicRepository {
```

Comment:
Not. :blush: 
Everybody who doesn't want to deal with Kafka is welcome to use it: Integration- & Acceptance Tests, local Nakadi-Developer, local Non-Nakadi-Developer, ...